editor/export_template_manager.cpp

```diff
@@ -404,9 +404,7 @@ bool ExportTemplateManager::_install_file_selected(const String &p_file, bool p_
 			// Read.
 			unzOpenCurrentFile(pkg);
 			ret = unzReadCurrentFile(pkg, data.ptrw(), data.size());
-			if (ret != UNZ_OK) {
-				break;
-			}
+			ERR_BREAK_MSG(ret < 0, (vformat(TTR("An error occured while attempting to read from file: %s. This file will not be used."), file)));
```

Comment:
Should be "occurred". We don't translate stdout error messages so you can remove the TTR. And there are unnecessary parentheses around the `vformat()`. (Applies to all 3 checks.)

Reply:
Amended, thank you!